buildSrc/src/main/kotlin/Library.kt

```diff
@@ -6,7 +6,7 @@ object Library {
 
     // CORE
 
-    private const val COROUTINES_VERSION = "1.5.0"
+    private const val COROUTINES_VERSION = "1.5.1"
```

Comment:
We can't update to this until Jetpack Compose updates to Kotlin `1.5.20`.

Reply:
Reverted.